requirements/edx/github.txt

```diff
@@ -13,6 +13,7 @@
 -e git+https://github.com/edx/django-wiki.git@cd0b2b31997afccde519fe5b3365e61a9edb143f#egg=django-wiki
 -e git+https://github.com/edx/django-oauth2-provider.git@0.2.7-fork-edx-5#egg=django-oauth2-provider
 -e git+https://github.com/edx/MongoDBProxy.git@25b99097615bda06bd7cdfe5669ed80dc2a7fed0#egg=mongodb_proxy
+-e git+https://github.com/edx/nltk.git@2.0.6#egg=nltk
```

Comment:
`-e` make pip install the dependency in "editable" mode: https://pip.pypa.io/en/stable/reference/pip_install.html#editable-installs I'm actually not sure what that means.